lib/server/ForwardClientRequest.js

```diff
@@ -1,31 +1,37 @@
 "use strict";
 
-var Promise = require("bluebird");
```

Comment:
you're right, that's not right. these files are completely unrelated. i'll see if i can fix it